src/master/concepts.md

```diff
@@ -1,109 +1,50 @@
+To increase security, immudb can be provisioned with a signing key to ensure non-repudiation of database states. The immudb server will subsequently sign the state on request from a client application.
+That means that an auditor or a third party client, for instance, could verify the authenticity of the returned current state.
 
 Immuclient and [immugw](https://github.com/codenotary/immugw) are shipped with auditor capabilities.
-To get the signed state in combination with the auditor, launch...
-* ...immuclient with auditor capabilities:
+To get the (signed) state in combination with the immuclient with auditor capabilities:
 ```bash
-immuclient audit-mode --audit-username {immudb-username} --audit-password {immudb-pw} --audit-signature validate
+immuclient audit-mode --audit-username {immudb-username} --audit-password {immudb-pw} --server-signing-pub-key {state-public-key}
 ```
-* ...with [immugw](https://github.com/codenotary/immugw) with auditor capabilities:
-```bash
-./immugw --audit --audit-username {immudb-username} --audit-password {immudb-pw} --audit-signature validate
-```
-
-</WrappedSection>
 
-<WrappedSection>
-
-## Item References
-
-Enables the insertion of a special entry that references another item.
-
-</WrappedSection>
-
-<WrappedSection>
-
-## Primary Index
-
-The primary index enables queries and search based on the **data key**.
+Check [tamper-proof operations](develop/operations.htm) and [running an auditor with immuclient](develop/auditor.html) paragraphs for additional details.
 
 </WrappedSection>
 
 <WrappedSection>
 
-## Secondary Index
+## Key value and SQL
 
-The secondary index enables queries and search based on the **data value**.
-
-</WrappedSection>
-
-<WrappedSection>
-
-## Streams
-Allows client server communication with streams of “delimited” []byte messages.
-
-</WrappedSection>
-
-<WrappedSection>
+Immudb can be used as a tamper-proof key value store or SQL database, with audit history capabilities. Within single immudb instance a user can have both types of databases, it is even possible to have KV and SQL withing single database.
 
-## Cryptographic signatures
+Key value is a foundation layer for SQL, meaning that SQL is using key value store capabilities underneath. 
 
-A signature (PKI) provided by the client can become part of the insertion process.
+Check [user quickstart](quickstart.html) for instructions on how to start working with KV or SQL. 
 
 </WrappedSection>
 
 <WrappedSection>
 
-## Authentication (transport)
+## Operation modes
 
-Integrated mTLS offers the best approach for machine-to-machine authentication, also providing communications security (entryption) over the transport channel.
+Immudb can be run as full database server with [replicas](operations/replication.html) or easily [embedded](develop/embedding.html) as a lightweight database into application.
```

Comment:
```suggestion
Immudb can be run as full database server with [replicas](operations/replication.html) or can be easily [embedded](develop/embedding.html) as a lightweight database into application.
```

Reply:
Also I think that we use lower-case for the first character of `immudb` even if it  starts at the beginning of a sentence.

Reply:
that one is actualy mixed, I prefer to use it like here, but it's subjective. @jeroiraz , wdyt ?

Reply:
I also prefer lower case